src/app/(protected)/account/settings/page.tsx

```diff
@@ -85,21 +130,26 @@ const Setting = () => {
                   </FormGroupField>
                 </FormGroup>
                 <FormGroup>
+                  <Description>
+                    {isFocused.password &&
+                      'Password should be 8 to 32 characters, including letter, number and special character'}
```

Comment:
we probably think of abstract this magic string, and make it shareable between different pages

Reply:
add share styled component